### PR TITLE
Update Geant4 to 11.1.2

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

**Comment by iarspider** [comment]
please test

File: geant4.spec [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 ## INCLUDE compilation_flags_lto
 %define use_vecgeom 0
-%define tag 35f304cde19a3e2dfe3bb941d4fc76c2fe45fa26
-%define branch cms/v%{realversion}
-%define github_user cms-externals
+#%define tag 35f304cde19a3e2dfe3bb941d4fc76c2fe45fa26
+#%define branch cms/v%{realversion}
+#%define github_user cms-externals
+%define tag cf4c6712c60cdf2b4d8799e34e176917c2de358d
+%define branch v%{realversion}
```
Comment (smuzaffar) [inline geant4.spec]: @iarspider , there is no `v%{realversion}` branch in official geant4 repo,  I would suggest to just use `master`

Comment (iarspider) [reply geant4.spec]: Done.

**Comment by iarspider** [comment]
please test

**Comment by smuzaffar** [comment]
+externals

unit test failure is not related to this change

**Pull request merged**